index.bs

```diff
@@ -65,16 +65,19 @@ beyond those described in the Generic Sensor API [[!GENERIC-SENSOR]].
 Model {#model}
 =====
 
-The orientation sensors's associated {{Sensor}} subclass is the {{DeviceOrientationSensor}} class.
+The device orientation sensors's associated {{Sensor}} subclass is the {{DeviceOrientationSensor}} class.
 
-Orientation sensor is a [=high-level=] sensor which is created through [=sensor-fusion=]
-of [=low-level=] motion sensors (accelerometer, gyroscope, magnetometer).
+The device orientation sensor is a [=high-level=] sensor which is created through [=sensor-fusion=]
+of the [=low-level=] motion sensors (accelerometer, gyroscope). The type of fusion (complementary
```

Comment:
> DeviceOrientation is normally what is called "Game rotation vector". 

Could you bring some references proving that?

As for Android, `TYPE_ROTATION_VECTOR` gets data from all three low-level sensors (gyro, magnetometer, accelerometer) and that fits for all use-cases. On Windows there is an analogue for it [SENSOR_TYPE_AGGREGATED_DEVICE_ORIENTATION](https://msdn.microsoft.com/en-us/library/windows/desktop/dd318987(v=vs.85).aspx).

`TYPE_GEOMAGNETIC_ROTATION_VECTOR` and `TYPE_GAME_ROTATION_VECTOR` are power [optimizations](https://opensignal.com/sensors/library/game_rotational_vector_sensor) of TYPE_ROTATION_VECTOR targeted for bearing and game scenarios correspondingly. Windows has no analogues for them, meaning that implementation of those will take much more time.

`TYPE_ROTATION_VECTOR` should suffice for all use-cases covered by `TYPE_GEOMAGNETIC_ROTATION_VECTOR` and `TYPE_GAME_ROTATION_VECTOR`

The `DeviceOrientationSensor` is gonna wrap  `TYPE_ROTATION_VECTOR` on Android and `SENSOR_TYPE_AGGREGATED_DEVICE_ORIENTATION` on Windows. Further we can add smth like `CompassSensor : DeviceOrientationSensor  // ignores gyro.`  to the same specification as an optimized class.